app/configurator/configurator-state/reducer.spec.tsx

```diff
@@ -700,8 +700,12 @@ describe("colorMapping", () => {
       chartConfig.fields.segment?.componentId ===
         "mapDataset___newAreaLayerColorIri"
     );
-    expect(chartConfig.fields.segment?.palette === "dimension");
-    expect(chartConfig.fields.segment?.colorMapping).toEqual({
+    expect(chartConfig.fields.color.paletteId === "dimension");
+    expect(
+      chartConfig.fields.color.type === "single"
+        ? chartConfig.fields.color.color
+        : chartConfig.fields.color.colorMapping
+    ).toEqual({
       orange: "rgb(255, 153, 0)",
     });
```

Comment:
yeah true, I will keep this in mind

Reply:
I think we will keep maps and tables as the are currently, just with a change of `paletteId`? If this is the case, then it would be good just to not forget about this (looks like this test is about "regular"color fields) – let me know if I missed something :)

Reply:
Ah so we don't intend to change table and map any further for this task?